project_type/models/project_type.py

```diff
@@ -21,6 +22,7 @@ class ProjectType(models.Model):
     project_ok = fields.Boolean(string="Can be applied for projects", default=True)
     task_ok = fields.Boolean(string="Can be applied for tasks")
     code = fields.Char(copy=False)
+    sequence = fields.Integer(string="Order")
```

Comment:
You should not change the display name. Order is not optimal because of Sales Order / Purchase Order.